zerver/webhooks/slack/tests.py

```diff
@@ -2,31 +2,71 @@
 
 from zerver.lib.test_classes import WebhookTestCase
 
+EXPECTED_TOPIC = "Message from Slack"
```

Comment:
This was previously marked for translation, but none of our other outgoing webhooks do that so I removed it. But maybe we want to keep that so as to not break that expectation for existing implementations?